github/github.go

```diff
@@ -392,21 +392,25 @@ func (client *Client) PullRequestForSHA(repo *github.Repository, SHA string) (*g
 		return nil, nil
 	}
 
+	if pullRequest.Mergeable != nil {
+		return pullRequest, nil
+	}
+
 	ticker := time.NewTicker(5 * time.Second)
 	for {
+		select {
+		case <-ticker.C:
```

Comment:
Oh, forgot to mention it before - this `select` isn't necessary, as there is only one condition. You can just do `<-ticker.C` to block until a tick arrives instead.